spec/controllers/state_file/archived_intake/archived_intake_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+describe StateFile::ArchivedIntakes::ArchivedIntakeController, type: :controller do
+  let(:ip_address) { '192.168.0.1' }
+  let(:email_address) { 'test@example.com' }
+  let(:request_instance) { instance_double(StateFileArchivedIntakeRequest) }
+
+  before do
+    allow(controller).to receive(:ip_for_irs).and_return(ip_address)
+    allow(session).to receive(:[]).with(:email_address).and_return(email_address)
+  end
+
+  describe '#current_request' do
+    it 'finds the StateFileArchivedIntakeRequest by IP and email address' do
+      expect(StateFileArchivedIntakeRequest).to receive(:find_by).with(
+        ip_address: ip_address,
+        email_address: email_address
+      ).and_return(request_instance)
+
+      expect(controller.current_request).to eq(request_instance)
```

Comment:
i think it seems unnecessary to expect the `find_by`. it would probably be sufficient to test that the return value of `current_request` is the right record.
```suggestion
      expect(controller.current_request).to eq(request_instance)
```